lib/simde/simde/x86/sse.h

```diff
@@ -3835,11 +3991,11 @@ simde_x_mm_setone_ps (void) {
   simde__m128 t = simde_mm_setzero_ps();
   return simde_mm_cmpeq_ps(t, t);
 }
-/*
+
```

Comment:
Likewise, does `simde_mm_sfence` still need to be commented out?

Reply:
yeah, without this MemoryBarrier causes a compiler error, I assume it's because of not including "windows.h" and me not using SIMDE_X86_SSE_NATIVE